src/runtime/nitro/renderers/satori/index.ts

```diff
@@ -11,7 +11,6 @@ import encoding from './plugins/encoding'
 import { fonts, satoriOptions } from '#nuxt-og-image/config'
 import loadSvg2png from '#nuxt-og-image/svg2png'
 import loadSatori from '#nuxt-og-image/satori'
-import { useRuntimeConfig } from '#internal/nitro'
```

Comment:
:information_source:  Some linting seems to have removed this line automatically.